etc/kayobe/ansible/build-ofed.yml

```diff
@@ -0,0 +1,111 @@
+---
+- name: Build OFED packages
+  hosts: ofed-builder
+  gather_facts: false
+  vars:
+    stackhpc_mlnx_ofed_file_string: MLNX_OFED_LINUX-{{ stackhpc_pulp_mlnx_ofed_version }}-rhel9.{{ stackhpc_pulp_repo_rocky_9_minor_version }}-x86_64
+  tasks:
+    - name: Install package dependencies
+      ansible.builtin.dnf:
+        name:
+          - std
+          - debootstrap
+          - kpartx
+          - cloud-init
+          - perl
+          - rpm-build
+          - automake
+          - patch
+          - kernel-devel
+          - autoconf
+          - pciutils
+          - kernel-rpm-macros
+          - lsof
+          - libtool
+          - tk
+          - gcc-gfortran
+          - tcl
+      state: present
+
+    - name: Install python dependencies
+      ansible.builtin.pip:
+        name: pulp-cli
+
+    - name: Create build directory
+      ansible.builtin.file:
+        path: /opt/ofed
+        state: directory
+        mode: 0777
+      become: true
+
+    - name: Download MellanoxOFED archive
+      ansible.builtin.get_url:
+        url: https://content.mellanox.com/ofed/MLNX_OFED-{{ stackhpc_pulp_mlnx_ofed_version }}/{{ stackhpc_mlnx_ofed_file_string }}.tgz
+        dest: /opt/ofed/ofed-archive
+
+    - name: Extract MellanoxOFED archive
+      ansible.builtin.unarchive:
+        src: /opt/ofed/ofed-archive
+        dest: /opt/ofed
+
+    - name: Ensure the current kernel is supported
+      ansible.builtin.shell:
+        cmd: |
+          /opt/ofed/{{ stackhpc_mlnx_ofed_file_string }}/mlnx_add_kernel_support.sh \
+          --mlnx_ofed /opt/ofed/{{ stackhpc_mlnx_ofed_file_string }} \
+          --make-tgz -y \
+      become: true
+
+    - name: Extract the new archive
+      ansible.builtin.unarchive:
+        src: /tmp/{{ stackhpc_mlnx_ofed_file_string }}-ext.tgz
+        dest: /opt/ofed
+      become: true
+
+    - name: Assign timestamp variable
+      ansible.builtin.set_fact:
+        stackhpc_pulp_repo_mlnx_ofed_version: "{{ ansible_date_time.iso8601_basic_short }}"
+
+    - name: Append timestamp to pulp-repo-versions
+      ansible.builtin.lineinfile:
+        path: "{{ lookup('env', KAYOBE_CONFIG_PATH) }}/pulp-repo-versions.yml"
+        line: "stackhpc_pulp_repo_mlnx_ofed_version: {{ stackhpc_pulp_repo_mlnx_ofed_version }}"
+
+    - name: Create Pulp repository for OFED
+      pulp.squeezer.rpm_repository:
+        pulp_url: "{{ stackhpc_release_pulp_url }}"
+        username: "{{ stackhpc_release_pulp_username }}"
+        password: "{{ stackhpc_release_pulp_password }}"
+        name: "{{ stackhpc_pulp_repo_mlnx_ofed.name }}"
+        state: present
+      retries: "{{ pulp_repository_rpm_repositories_retries }}"
```

Comment:
Is pulp_repository_rpm_repositories_retries defined here?

Reply:
Defaults to 3, Not sure if we should just hardcode that or leave pulp_timeout_retries to be defined when required.